TradeItIosTicketSDK2/TradeItPortfolioPositionsTableViewManager.swift

```diff
@@ -71,32 +72,97 @@ class TradeItPortfolioPositionsTableViewManager: NSObject, UITableViewDelegate,
     }
     
     func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-        var cell: UITableViewCell!
         let position = self.positions[indexPath.row]
+        let cell = self.provideCell(forTableView: tableView,
+                                    forPortfolioPosition: position,
+                                    selected: self.selectedPositionIndex == indexPath.row)
+
+        return cell
+    }
+
+    func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
```

Comment:
See if there is a way to calculate these heights based on the storyboard views instead of just hardcoding them.